frontend/src/utils/wildcard.js

```diff
@@ -39,14 +41,32 @@ export function wildcardObjectsFromStrings (wildcardStrings) {
       endsWithWildcard,
       customWildcard,
       isWildcard: startsWithWildcard || endsWithWildcard || customWildcard,
-      regex: new RegExp('^' + pattern + '$')
+      test: function (value) {
+        if (!this.isWildcard) {
+          return value === this.originalValue
```

Comment:
```suggestion
          return value === this.value
```
We don't need the originalValue